snapshot/exporter/fs/fs.go

```diff
@@ -70,12 +70,10 @@ func (p *FSExporter) StoreFile(pathname string, fp io.Reader) error {
 		f.Close()
 		return err
 	}
-	if err := f.Sync(); err != nil {
-		//logging.Warn("sync failure: %s: %s", pathname, err)
-	}
-	if err := f.Close(); err != nil {
-		//logging.Warn("close failure: %s: %s", pathname, err)
-	}
+	_ = f.Sync()
+	//if err: logging.Warn("sync failure: %s: %s", pathname, err)
+	_ = f.Close()
+	//if err: logging.Warn("close failure: %s: %s", pathname, err)
```

Comment:
meh; either fix the error handling or remove the comments.

Reply:
fixed